integration_tests/examples.spec.ts

```diff
@@ -1,5 +1,6 @@
 import { execFile } from "child_process";
-import { readFileSync, unlinkSync, writeFileSync } from "fs";
+import { readdirSync, readFileSync, unlinkSync, writeFileSync } from "fs";
+import { shuffle } from "underscore";
```

Comment:
Is there a specific reason you used the underscore?
The lodash also has the shuffle method.

Reply:
Thank you. I didn't know that. I'll fix it.